src/components/CourseContainer/index.tsx

```diff
@@ -1,24 +1,38 @@
-import React, { useContext } from 'react';
+import React, { useContext, useState } from 'react';
 import ago from 's-ago';
 
 import { Button, Course, CourseAdd } from '..';
 import { ScheduleContext } from '../../contexts';
+import CourseNavMenu from '../CourseNavMenu';
 
 import 'react-virtualized/styles.css';
 import './stylesheet.scss';
 
 export default function CourseContainer(): React.ReactElement {
   const [{ oscar, desiredCourses }] = useContext(ScheduleContext);
+  const courseTabs = ['Courses', 'Recurring Events'];
+  const [currentTab, setCurrentTab] = useState(0);
 
   return (
     <div className="CourseContainer">
-      <div className="scroller">
-        <div className="course-list">
-          {desiredCourses.map((courseId) => {
-            return <Course courseId={courseId} key={courseId} />;
-          })}
-        </div>
-        <CourseAdd className="course-add" />
+      <div className="CourseNavMenu">
+        <CourseNavMenu
+          items={courseTabs}
+          currentItem={currentTab}
+          onChangeItem={setCurrentTab}
+        />
+        {courseTabs[currentTab] === courseTabs[0] ? (
+          <div className="scroller">
+            <div className="course-list">
+              {desiredCourses.map((courseId) => {
+                return <Course courseId={courseId} key={courseId} />;
+              })}
+            </div>
+            <CourseAdd className="course-add" />
+          </div>
+        ) : (
+          <div />
```

Comment:
Turn this div to `<div className="scroller"/>` for now as a placeholder so the pane doesn't resize when we switch between tabs